src/App.js

```diff
@@ -1,25 +1,37 @@
-import logo from './logo.svg';
+/* eslint-disable react-hooks/exhaustive-deps */
+import React, { useEffect } from 'react';
+import { Route, Routes } from 'react-router-dom';
+import { useDispatch } from 'react-redux';
+import { showMissions } from './Redux/Missions/Missions';
+import NavBar from './Components/NavBar';
+import Rockets from './Pages/Rockets';
+import Missions from './Pages/Missions';
+import Profile from './Pages/Profile';
+
 import './App.css';
 
 function App() {
+  const dispatch = useDispatch();
+  useEffect(() => {
+    dispatch(showMissions());
+  }, []);
```

Comment:
- [ ] As demanded by the project requirements, the missions data should only be fetched (once) when a user navigates to the Missions section. I can see that you are fetching the data when the App is rendered. Kindly fix this please.

Reply:
we've Fixed this as well 🍾